Content.Client/Lobby/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -770,6 +775,22 @@ public void RefreshTraits()
             }
         }
 
+        private bool AllowSpecie(SpeciesPrototype specie) // Starlight
```

Comment:
Rename it to race. Species can be only in the plural.
```suggestion
        private bool AllowRace(SpeciesPrototype race) // Starlight
```

Reply:
fixed